### PR TITLE
Add @since on Util.isSafeToRedirectTo()

### PR DESCRIPTION
**Pull request by batmat**
@reviewbybees

cc @daniel-beck

### CODE REVIEW

File: core/src/main/java/hudson/Util.java [hunk 1]
```diff
@@ -1597,7 +1597,8 @@ public static boolean isAbsoluteUri(@Nonnull String uri) {
     }
 
     /**
-     * Return true iff the parameter does not denote an absolute URI and not a scheme-relative URI.
+     * Return true if the parameter does not denote an absolute URI and not a scheme-relative URI.
+     * @since 2.3 / 1.651.2
```
Comment (oleg-nenashev) [inline core/src/main/java/hudson/Util.java]: Did we backport unrestricted API to 1.651.x? :(

Comment (olivergondza) [inline core/src/main/java/hudson/Util.java]: It is actually new API we needed to introduce for a security fix (https://github.com/jenkinsci/jenkins/commit/2ed0c046dfbb2003a17df27c53777e72c6eaff25, https://github.com/jenkinsci/jenkins/commit/e6efae7d0d742f068498366424b0059e9ed42e4c)

**Comment by batmat** [comment]
\o/ :blush: 
@daniel-beck now the stupidness is fixed, does the @since look correct to you?

**Comment by oleg-nenashev** [comment]
I would remove 1.651.2 at all in order to avoid confusing people

**Comment by daniel-beck** [comment]
@batmat Looks good.

@oleg-nenashev It's not like there's no precedence for this:

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/UpdateSite.java#L618
https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/UpdateSite.java#L618
https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/ParametersAction.java#L110

**Review by oleg-nenashev — APPROVED** [review]
OK. I'm fine with it as is

**Pull request merged**